src/mlpack/methods/ann/visitor/copy_visitor.hpp

```diff
@@ -26,7 +26,7 @@ class CopyVisitor : public boost::static_visitor<LayerTypes>
 {
  public:
   template <typename LayerType>
-  LayerTypes operator () (LayerType* ) const;
+  LayerTypes operator () (LayerType*) const;
```

Comment:
I think in other places for operators we've done, e.g., `operator=(args& args...)`, so I would expect here it might be `operator()(LayerType*)`; is this a gray area in the style check rules?